.ci/resolve-kata-dependencies.sh

```diff
@@ -56,8 +56,8 @@ apply_depends_on() {
 		dependency_branch="p${pr_id}"
 		git fetch origin "pull/${pr_id}/head:${dependency_branch}" && \
 			git checkout "${dependency_branch}" && \
-			git rebase "origin/${branch}"
-			# And show what we rebased on top of to aid debugging
+			git merge "origin/${branch}"
```

Comment:
Could we make this `--ff-only`? Would that work for https://github.com/kata-containers/kata-containers/pull/330 ?

Reply:
We don't need to require --ff-only either. Just simulate a github merge action is enough. And yes, I think this should work for kata-containers/kata-containers#330. waiting to verify.